docs/volto/bootstrap.md

```diff
@@ -1,31 +1,41 @@
 ---
 myst:
   html_meta:
-    "description": ""
-    "property=og:description": ""
-    "property=og:title": ""
-    "keywords": ""
+    "description": "Setting up backend and frontend"
+    "property=og:description": "Setting up backend and frontend"
+    "property=og:title": "Bootstrapping A Volto project"
+    "keywords": "Plone, Volto"
 ---
 
 (bootstrap-volto-label)=
 
-# Bootstrapping A Volto Project
+# Bootstrapping A Volto project
+
+```{note}
+It is recommended to work on a Linux or Mac.
+```
 
-## Installing Plone
+
+## Installing Plone backend
 
 In order to run Volto you need a backend.
 This can be either Plone or Guillotina.
-For this course we will use Plone, you can download Plone at <https://plone.org/download>.
-We need plone.restapi, so make sure you have that installed and configured correctly.
-For an example look into the api folder of the Volto repository: <https://github.com/plone/volto/tree/master/api>
-
-```{warning}
-Make sure you set a CORS policy or things tend to magically go wrong. See <https://github.com/plone/volto/blob/master/api/buildout.cfg> for an example.
+For this course we will use Plone.
+Because we will focus on the frontend, for simplicity we install a backend as docker container.
+The installation of a Plone backend as docker container is being described at `docs.plone.org`: {ref}`plone6docs:install-containers-label`.
+ 
+In short: Create a directory `backend` and run 
+
+```{code-block} shell
```

Comment:
`code-block` is only needed when you want to get fancy with reStructuredText features, such as line numbers and emphasizing lines.

````suggestion
```shell
````